Makefile

```diff
@@ -70,6 +70,19 @@ test-python-integration:
 test-python-universal-contrib:
 	PYTHONPATH='.' FULL_REPO_CONFIGS_MODULE=sdk.python.feast.infra.offline_stores.contrib.contrib_repo_configuration FEAST_USAGE=False IS_TEST=True python -m pytest -n 8 --integration --universal sdk/python/tests
 
+test-python-universal-postgres:
+	PYTHONPATH='.' \
+		FULL_REPO_CONFIGS_MODULE=sdk.python.feast.infra.offline_stores.contrib.contrib_repo_configuration \
+		FEAST_USAGE=False \
+		IS_TEST=True \
+		python -m pytest --integration --universal \
+			-k "not test_historical_retrieval_fails_on_validation and \
```

Comment:
There were some tests that store the historical result. I have not implemented anything for that because I find it strange to create a lot of tables without a way to delete them when done. I guess the stored historical retrieval could simply be the SQL query rather than a table name.
But I'll take a look at the ones that I'm omitting and make sure that I'm not omitting tests that I shouldn't be.